test/metabase/cmd/remove_encryption_test.clj

```diff
@@ -0,0 +1,38 @@
+(ns metabase.cmd.remove-encryption-test
+  (:require
+   [clojure.java.jdbc :as jdbc]
+   [clojure.test :refer :all]
+   [metabase.cmd :as cmd]
+   [metabase.cmd.remove-encryption :refer [remove-encryption!]]
+   [metabase.db :as mdb]
+   [metabase.test :as mt]
+   [metabase.test.fixtures :as fixtures]
+   [metabase.util.encryption :as encryption]
+   [metabase.util.encryption-test :as encryption-test]
+   [toucan2.core :as t2]))
+
+(set! *warn-on-reflection* true)
+
+(use-fixtures :once (fixtures/initialize :db))
+
+(defn- raw-value [data-source keyy]
+  (:value (first (jdbc/query {:connection data-source}
+                             ["select \"VALUE\" from setting where setting.\"KEY\"=?;" keyy]))))
```

Comment:
nit - up to you, but this is the kind of thing I personally find more readable with a threading macro
```suggestion
  (-> (jdbc/query {:connection data-source}
                  ["select \"VALUE\" from setting where setting.\"KEY\"=?;" keyy])
      first
      :value))
```